tensorboard/components/tf_data_selector/tf-data-select-row.ts

```diff
@@ -0,0 +1,97 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+namespace tf_data_selector {
+
+Polymer({
+  is: 'tf-data-select-row',
+  properties: {
+    removable: Boolean,
```

Comment:
What's the significance of this property?  It doesn't seem to be used right now, unless it's something Polymer has special handling for?